virtualizarr/tests/__init__.py

```diff
@@ -46,15 +47,14 @@ def create_manifestarray(
     The manifest is populated with a (somewhat) unique path, offset, and length for each key.
     """
 
-    zarray = ZArray(
+    zarray = ArrayV2Metadata(
         chunks=chunks,
-        compressor="zlib",
+        compressor={"id": "zlib"},
```

Comment:
Is the presence of `"id"` a v2 vs v3 difference? because you don't seem to use it in the other tests

Reply:
In V3, the `codecs` array is made up of `(name, configuration)` objects, but in V2 the `id` field serves as the name. In this particular test, I just chose to convert the `ZArray` to an `ArrayV2Metadata`

Actually please ignore any of the changes in `tests` for now because I will need to edit almost every test to excise `ZArray` :) The few changes I added were done up until I hit the `kerchunk` module import issue